287_Find_the_Duplicate_Number.py

```diff
@@ -0,0 +1,17 @@
+# n = len(nums)
+# time: O(n)
+# space: O(1)
+
+from typing import List
+
+
+
+class Solution:
+    def findDuplicate(self, nums: List[int]) -> int:
+
+        count = {}
```

Comment:
This variable named count is a dictionary, so you should call it "(words of a key)_(words of a value)".